dives/TestAtmPress.xml

```diff
@@ -0,0 +1,1385 @@
+<divelog program='subsurface' version='3'>
+<settings>
+  <autogroup state='1' />
+</settings>
+<divesites>
+<site uuid='4fdcf19b' name='Loc1'>
+</site>
+<site uuid='b15ad032' name='Loc2'>
+</site>
+</divesites>
+<dives>
+<trip date='2019-04-09' time='09:15:16' location='Scottburgh'>
+<dive number='524' tripflag='NOTRIP' divesiteid='b15ad032' date='2019-04-09' time='09:15:16' airpressure='1.012 bar' duration='42:08 min'>
+  <cylinder size='10.0 l' workpressure='232.0 bar' description='10ℓ 232 bar' o2='30.0%' depth='42.96 m' />
+  <cylinder size='10.0 l' workpressure='232.0 bar' description='10ℓ 232 bar' o2='30.0%' start='200.0 bar' end='113.0 bar' depth='42.96 m' />
+  <weightsystem weight='3.0 kg' description='belt' />
+  <divecomputer model='Uwatec Galileo Trimix' deviceid='462769ce' diveid='14106bf6'>
+  <depth max='30.068 m' mean='21.394 m' />
+  <temperature air='24.6 C' water='23.2 C' />
+  <water salinity='1025 g/l' />
+  <event time='0:00 min' type='25' flags='1' name='gaschange' cylinder='0' o2='30.0%' />
+  <event time='18:00 min' type='25' flags='2' name='gaschange' cylinder='1' o2='30.0%' />
+  <event time='33:36 min' type='25' flags='1' name='gaschange' cylinder='0' o2='30.0%' />
+  <event time='34:17 min' type='8' name='Deepstop' />
+  <event time='37:36 min' type='8' name='bookmark' />
+  <sample time='0:00 min' depth='0.0 m' temp='24.4 C' rbt='99:00 min' />
+  <sample time='0:04 min' depth='2.439 m' temp='24.8 C' rbt='99:00 min' />
+  <sample time='0:08 min' depth='3.395 m' temp='24.4 C' rbt='99:00 min' />
+  <sample time='0:12 min' depth='3.902 m' temp='24.8 C' pressure='208.5 bar' rbt='99:00 min' />
+  <sample time='0:16 min' depth='4.488 m' pressure='208.25 bar' rbt='98:00 min' />
+  <sample time='0:20 min' depth='5.795 m' rbt='97:00 min' />
```

Comment:
This adds ~1300 lines of samples... but I don't think those samples are really relevant or interesting for the purpose of this test. Wouldn't it make more sense to have just 5 samples for a simple profile and make it easy to spot what matters here, the `airpressure`?